db/models/assessment_record/allocation_association.py

```diff
@@ -0,0 +1,26 @@
+from db import db
+from flask_sqlalchemy.model import DefaultMeta
+from sqlalchemy import Column
+from sqlalchemy import ForeignKey
+from sqlalchemy import func
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy.dialects.postgresql import UUID
+
+
+BaseModel: DefaultMeta = db.Model
+
+
+class AllocationAssociation(BaseModel):
```

Comment:
Can we add why we adding this model as a comment

Reply:
Sure, done.